code/modules/antagonists/fugitive/hunter.dm

```diff
@@ -0,0 +1,166 @@
+			to_chat(owner, "<B>The criminals should be on the station, we have special huds implanted to recognize them.</B>")
+			to_chat(owner, "<B>As we have lost pretty much all power over these damned lawless megacorporations, it's a mystery if their security will cooperate with us.</B>")
+		if("russian")
+			to_chat(src, "<span class='danger'>Ay blyat. I am a space-russian smuggler! We were mid-flight when our cargo was beamed off our ship!</span>")
+			to_chat(src, "<span class='danger'>We were hailed by a man in a green uniform, promising the safe return of our goods in exchange for a favor:</span>")
+			to_chat(src, "<span class='danger'>There is a local station housing fugitives that the man is after, he wants them returned; dead or alive.</span>")
+			to_chat(src, "<span class='danger'>We will not be able to make ends meet without our cargo, so we must do as he says and capture them.</span>")
+
+	to_chat(owner, "<span class='boldannounce'>You are not an antagonist in that you may kill whomever you please, but you can do anything to ensure the capture of the fugitives, even if that means going through the station.</span>")
+	owner.announce_objectives()
+
+/datum/antagonist/fugitive_hunter/create_team(datum/team/fugitive_hunters/new_team)
+	if(!new_team)
+		for(var/datum/antagonist/fugitive_hunter/H in GLOB.antagonists)
+			if(!H.owner)
+				continue
+			if(H.hunter_team)
+				hunter_team = H.hunter_team
+				return
+		hunter_team = new /datum/team/fugitive_hunters
+		hunter_team.backstory = backstory
+		hunter_team.update_objectives()
+		return
+	if(!istype(new_team))
+		stack_trace("Wrong team type passed to [type] initialization.")
+	hunter_team = new_team
+
+/datum/antagonist/fugitive_hunter/get_team()
+	return hunter_team
+
+/datum/team/fugitive_hunters
+	var/backstory = "error"
+
+/datum/team/fugitive_hunters/proc/update_objectives(initial = FALSE)
+	objectives = list()
+	var/datum/objective/O = new()
+	O.team = src
+	objectives += O
+
+/datum/team/fugitive_hunters/proc/assemble_fugitive_results()
+	var/list/fugitives_counted = list()
+	var/list/fugitives_dead = list()
+	var/list/fugitives_captured = list()
+	for(var/datum/antagonist/fugitive/A in GLOB.antagonists)
+		if(!A.owner)
+			continue
+		fugitives_counted += A
+		if(A.owner.current.stat == DEAD)
+			fugitives_dead += A
+		if(A.is_captured)
+			fugitives_captured += A
+	. = list(fugitives_counted, fugitives_dead, fugitives_captured) //okay, check out how cool this is.
+
+/datum/team/fugitive_hunters/proc/all_hunters_dead()
+	var/dead_boys = 0
+	for(var/I in members)
+		var/datum/mind/hunter_mind = I
+		if(!(ishuman(hunter_mind.current) || (hunter_mind.current.stat == DEAD)))
+			dead_boys++
+	return dead_boys >= members.len
+
+/datum/team/fugitive_hunters/proc/get_result()
+	var/list/fugitive_results = assemble_fugitive_results()
+	var/list/fugitives_counted = fugitive_results[1]
+	var/list/fugitives_dead = fugitive_results[2]
+	var/list/fugitives_captured = fugitive_results[3]
```

Comment:
> generates 3 lists from a list from a proc that's 3 different lists put into a list

head hurts

Reply:
![image](https://user-images.githubusercontent.com/40974010/53837695-11c79b00-3f48-11e9-9d77-6997a9570f2c.png)
i'm assuming (and hoping) head hurts in the complicated way not the bad way